bin/pycbc_multi_inspiral

```diff
@@ -197,20 +200,33 @@ def get_weighted_antenna_patterns(Fp_dict, Fc_dict, sigma_dict):
     return wp, wc
 
 
-def get_projection_matrix(wp, wc):
+def get_projection_matrix(wp, wc, projection='standard'):
     """
     Output: projection_matrix: Projects the data onto the signal space
     Input:  wp,wc: The weighted antenna response fuctions to plus and cross
                    polarisations respectively
+            projection_type: standard, left or right
```

Comment:
The kwarg here is `projection` not `projection_type`. Can you change the doc string?